hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCommitRequestWithFSO.java

```diff
@@ -268,10 +264,20 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, Execut
         // Otherwise, it causes data loss once those shared blocks are added
         // to deletedTable and processed by KeyDeletingService for deletion.
         filterOutBlocksStillInUse(omKeyInfo, oldVerKeyInfo);
-
+        long totalSize = 0;
+        long totalNamespace = 0;
```

Comment:
Same deal as above. I believe you mentioned you are still working on KeyCommitRequest

Reply:
done